### ISSUE TITLE
JSON message in TypeScript

### ISSUE THREAD

**Issue by angelovdev** [nfriedly/express-rate-limit#262]
I am trying to return a custom JSON when the limit is exceeded but I get this error:

`Type '{ error: string; }' is missing the following properties from type 'Message': status, message ts(2322)`

This is the code I've tried as per a previous issue:
```js
message: {
  error: "Too many accounts created from this IP, please try again after an hour"
}
```

**Comment by nfriedly** [comment]
Alright, I think https://github.com/DefinitelyTyped/DefinitelyTyped/pull/57394 will fix that, as long as it doesn't get thrown out for not following all of their guidelines.

**Comment by nfriedly** [comment]
Would you mind testing that PR and adding a comment to it with your results?

**Comment by nfriedly** [comment]
We just finished the full conversion to typescript and published it to npm in v6.0.0. Pleas try out that version and let me know if it works for you.

**Comment by angelovdev** [comment]
Will do that when I have some time available
